src/Command/BakeMigrationCommand.php

```diff
@@ -76,8 +76,21 @@ public function templateData(Arguments $arguments): array
             $pluginPath = $this->plugin . '.';
         }
 
+
+        $arguments = $arguments->getArguments();
+        unset($arguments[0]);
+        $columnParser = new ColumnParser();
+        $fields = $columnParser->parseFields($arguments);
+        $indexes = $columnParser->parseIndexes($arguments);
+        $primaryKey = $columnParser->parsePrimaryKey($arguments);
+
         $action = $this->detectAction($className);
 
+        if (empty($action) && count($fields)) {
+            /** @psalm-suppress PossiblyNullReference */
+            $this->io->abort('Migration name should start with one of the following prefixes: Create, Drop, Add, Remove, Alter. See: https://book.cakephp.org/migrations/4/en/index.html#migrations-file-name');
+        }
+
         if (empty($action)) {
```

Comment:
Same here I guess

Reply:
well. That is the original code as it is now in the 4.x branch.
Should I change it?

Reply:
Ahyou ment the message not replacing `empty()`. Done